challenge3/webapp/y-shinada/app/forms/highchart/index_form.rb

```diff
@@ -0,0 +1,37 @@
+class Highchart::IndexForm
+  TRACKS = [:track_from, :track_to].freeze
+
+  attr_reader :params
+
+  def initialize(params)
+    @params = params
+  end
+
+  def city
+    @city ||= (params[:city] || "")
+  end
+
+  def cap_city
+    @cap_city ||= city.capitalize
+  end
+
+  def num_of_people
+    @num_of_people ||= params[:num_of_people].to_i
+  end
+
+  TRACKS.each do |k|
+    define_method k do
+      return if params[k].blank?
+
+      ivar_key = "@#{k}".to_sym
```

Comment:
Object#instance_variable_get は文字列を引数にとれるので
`.to_sym` する必要はない。
https://ruby-doc.org/core-2.6/Object.html#method-i-instance_variable_get

```suggestion
      ivar_key = "@#{k}"
```